modules/java_api/samples/face_detection_sample_async/Main.java

```diff
@@ -232,12 +236,13 @@ public void run() {
                     Imgproc.putText(img, label1, new Point(10, 80), 0, 0.7, color, 1);
                 }
                 HighGui.imshow("Detection", img);
+                HighGui.waitKey(1);
             }
 
             captureThread.join();
             inferThread.join();
 
-            HighGui.waitKey(0);
+            HighGui.waitKey(1);
```

Comment:
Not sure if we really need `waitKey` here

Reply:
Still have a question why we need `waitKey` here?

Reply:
Can't destroy window without this `waitKey`